### PR TITLE
grafanastats macro: better detection if grafana is reachable or not

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

**Comment by muhamadazmy** [comment]
looks fine to me

File: apps/gridportal/base/Cockpit/.macros/page/grafanastats/1_grafanastats.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 def main(j, args, params, tags, tasklet):
     page = args.page
     grid = "<iframe id='ifr' width='100%' height='600' src='/grafana/dashboard/db/nodes-stats?var-node=All&theme=light'></iframe>"
-    grafana = j.portal.server.active.cfg['grafana']
-    if not j.sal.nettools.checkUrlReachable('http://localhost/grafana/'):
+    host = j.portal.server.active.cfg['grafana']['host']
+    port = int(j.portal.server.active.cfg['grafana']['port'])
```
Comment (grimpy) [inline apps/gridportal/base/Cockpit/.macros/page/grafanastats/1_grafanastats.py]: is grafana hard dependency of cockpit? How we garantee cfg['grafana'] is always set?

Comment (rkhamis) [inline apps/gridportal/base/Cockpit/.macros/page/grafanastats/1_grafanastats.py]: Good point. We need to figure out a more resilient way...
It IS a hard dependency, but components get installed in lots of different ways and cfg could be missing.

I would just make sure the grafana cfg is in the portal cfg and if it's not, give a message that grafana is missing

**Comment by grimpy** [comment]
Not relevant for this repo anymore should be moved to jscockpit if still needed

**Pull request closed without merging**